docs/CHILI-GraFx/users/intro/index.md

```diff
@@ -1,79 +1,78 @@
-
-### Permissions
+### Access
 
 !!! Definition
-	The authorization to perform an operation over a resource.
+    The authorization to perform an operation over a resource.
 
-In order to be able to perform an action over a resource, the user must have a role assigned to them that contains the proper permission to do so.
+In order to perform an action over a resource, the user must have a role assigned to them that contains the proper permission to do so.
 
-Permissions in CHILI GraFx not assigned on an individual user, but are assigned to a *Role*.
+Permissions in CHILI GraFx are not assigned on an individual user basis, but are assigned to a *Role*.
 
 **Examples**
 
-create, read, update, delete an asset
-
-create, read, update, delete a user
-
-create, remove an environment
+- Create, read, update, delete an asset
+- Create, read, update, delete a user
+- Create, remove an environment
 
 ### Roles
 
 !!! Definition
-	A set of permissions that eases the management of permissions to users or groups.
+    A set of permissions that eases the management of permissions for users or groups.
  
-Normally users are going to have more than just a single permission. It is also likely that you may want to be able to update the permissions of multiple users simultaneously. Roles make this possible.
+Normally users will have more than just a single permission. It is also likely that you may want to update the permissions of multiple users simultaneously. Roles make this possible.
 
 A Role is a container for permissions. Roles should contain all the necessary permissions needed for an intended purpose or workflow.
 
-
 ### CHILI GraFx Users
 
 !!! Definition
-	An individual with access to GraFx.
+    A person or system identified by a username that performs an operation over a resource
+
+We define a CHILI GraFx User as a person or a system. In many cases, you will interact as a human with the CHILI GraFx UI. 
+
+In some cases, you'll also need a system user, which can be used to interact with the CHILI GraFx APIs. It's wise to unlink this system user from a person. In case the human user changes jobs or roles, you don't need to redefine the access or role for the system user.
 
-A user is someone that has credentials to GraFx. A user can have 1 or more roles assigned to them. These roles will dictate the permissions that the user have which in turn controls their ability to view, create, update, and destroy resources in GraFx
+### User Groups
 
+User Groups in CHILI GraFx represent a collection of users who share common access needs and roles. Each user group is defined by the access and roles, which collectively determine the actions members of the group can perform in the system. By categorizing users into groups, you simplify access management, allowing administrators to efficiently assign and modify access rights for multiple users simultaneously, enhancing both security and operational efficiency.
```

Comment:
@BramVerniest : I would clarify that a User Group is a concept within 1 subscription, but not accross multiple subscriptions. So a user group can give you access to multiple environments within a subscription, but never to environments of multiple subscriptions.